src/provider/Provider.js

```diff
@@ -65,7 +73,7 @@ export default function Provider(logger) {
 
   // we are going to reimplement a simpler mechanism to call the transformers
   function execute(name, params, context) { // eslint-disable-line no-unused-vars
-    return {};
+    return transformerMap[name](params, context);
```

Comment:
Should probably check whether transformerMap[name] exist, to avoid getting errors like:
TypeError: undefined is not a function